### PR TITLE
Add a new method install_with_pip in generic.general_methods

### PR DESCRIPTION
**Pull request by maxcapodi78**
The methods is useful when pyaedt is used from AEDT console

### CODE REVIEW

**Review by MaxJPRey — APPROVED** [review]
LGTM.

File: pyaedt/generic/general_methods.py [hunk 1]
```diff
@@ -1574,6 +1574,48 @@ def _check_installed_version(install_path, long_version):
     return False
 
 
+def install_with_pip(package_name, package_path=None, upgrade=False, uninstall=False):  # pragma: no cover
+    """Install a new package using pip.
+    Useful to install a package from AEDT Console without launching the python environment.
```
Comment (PipKat) [inline pyaedt/generic/general_methods.py:1579]:
```suggestion
    This method can be used to install a package from the AEDT Console without launching the Python environment.
```

File: pyaedt/generic/general_methods.py [hunk 2]
```diff
@@ -1574,6 +1574,48 @@ def _check_installed_version(install_path, long_version):
...
+
+    Parameters
+    ----------
+    package_name : str
+        Name of the package to install.
+    package_path : str, optional
+        Package path for GitHub download. Example ``git+https://......``.
```
Comment (PipKat) [inline pyaedt/generic/general_methods.py:1586]:
```suggestion
        Path for the GitHub package to download. For example, ``git+https://.....``.
```

File: pyaedt/generic/general_methods.py [hunk 3]
```diff
@@ -1574,6 +1574,48 @@ def _check_installed_version(install_path, long_version):
...
+    upgrade : bool, optional
+        Whether to upgrade the package or not.
+    uninstall : bool, optional
+        Whether to install the package or uninstall the package.
```
Comment (PipKat) [inline pyaedt/generic/general_methods.py:1590]:
```suggestion
        Whether to install the package or uninstall the package.
```
This is confusing. The default is ``False``, but what does this do if you are choosing between installing or unintalling?

Comment (maxcapodi78) [reply pyaedt/generic/general_methods.py:1590]: 2 options: install a package or uninstall a package. True, uninstall, False install

**Pull request merged**